BookAndCanvas/SQL/setupDB.sql

```diff
@@ -0,0 +1,132 @@
+create database BookAndCanvas
+
+create table Users
+(
+	[Id] int primary key identity(1,1),
+	[FName] nvarchar(50) not null,
+	[LName] nvarchar(50) not null,
+	[Phone] nvarchar(50) not null,
+	[Email] nvarchar(50) not null,
+	[ImgUrl] nvarchar(50),
+	[Bio] nvarchar (max)
+)
+
+create table UserMsg
+(
+	[Id] int primary key identity(1,1),
+	[MessageId] int not null,
+	[BuyerId] int not null,
+	[SellerId] int not null
+)
+
+alter table UserMsg
+add constraint FK_UserMsg_User_Buyer 
+	foreign key (BuyerId)
+	references Users (Id)
+
+alter table UserMsg
+add constraint FK_UserMsg_User_Seller 
+	foreign key (SellerId)
+	references Users (Id)
+
+create table Message
+(
+	[Id] int primary key identity(1,1),
+	[MessageId] int not null,
+	[MsgDate] DateTime not null,
+	[isRead] bit not null,
+	[MsgTitle] nvarchar(250),
+	[MsgText] nvarchar(max)
+)
+
+alter table UserMsg
+add constraint FK_UserMsg_Message_Id 
+	foreign key (MessageId)
+	references Message (Id)
+
+create table Invoice
+(
+	[Id] int primary key identity(1,1),
+	[Date] DateTime not null,
```

Comment:
Invoice- I think we should use a default date method.  `DEFAULT GETDATE() `? This way we don't have to manually insert a date every time a record gets created.

Reply:
I don't know how to do this. Can you give me an example of how this is used?